packages/cli/test/helpers/init.js

```diff
@@ -1,5 +1,6 @@
 const path = require('path')
 process.env.TS_NODE_PROJECT = path.resolve('test/tsconfig.json')
+process.env.IS_HEROKU_TEST_ENV = 'true'
```

Comment:
This was added to account for no longer being an option [`cli.raiseErrors = true`](https://github.com/heroku/cli/blob/main/packages/apps-v5/src/commands/dashboard.js#L99)